core/org.eclipse.cdt.ui/src/org/eclipse/cdt/ui/newui/AbstractPage.java

```diff
@@ -168,44 +168,22 @@ public abstract class AbstractPage extends PropertyPage implements IPreferencePa
 	 * Bookeeping variables
 	 */
 	/**
-	 * @deprecated This field was never meant to be API. This field will be made private.
+	 * @noreference This field is not intended to be referenced by clients.
 	 */
-	@Deprecated(forRemoval = true)
 	protected boolean noContentOnPage = false;
 	/**
-	 * @deprecated This field was never meant to be API. This field will be made private.
+	 * @noreference This field is not intended to be referenced by clients.
 	 */
-	@Deprecated(forRemoval = true)
 	protected boolean displayedConfig = false;
```

Comment:
Shouldn't this field be private?

Reply:
Its accessed from AbstractSinglePage so I marked it as `@noreference` instead which gives us freedom to refactor in the future.